src/main/java/com/fcgl/madrid/points/controller/UserTrophyController.java

```diff
@@ -23,4 +32,13 @@ public void setPostService(UserTrophyService userTrophiesService) {
     public List<UserTrophy> findAll() {
         return userTrophiesService.findAll();
     }
+    // 
+    // public List<UserTrophy> findAll() {
+    //     return userTrophyRepository.findAll();
+    // }
+
+    @GetMapping(value="/fallback")
```

Comment:
Same as previous comment, this shouldn't go here. Should add a controller on `dev` package if you want to add this